common/djangoapps/third_party_auth/tests/specs/test_azuread.py

```diff
@@ -35,8 +35,8 @@ def setUp(self):
         'aud': 'abcdefgh-1234-5678-900a-0aa0a00aa0aa',
         'tid': 'abcdefgh-1234-5678-900a-0aa0a00aa0aa',
         'amr': ['pwd'],
-        'unique_name': 'email_value@example.com',
-        'upn': 'email_value@example.com',
+        'unique_name': 'user@email.com',
+        'upn': 'user@email.com',
```

Comment:
Why do these values need to be changed?

Reply:
This test is used to make sure that if a learner already exists in the system (collisions) it is handled appropriately and since we are checking both email and username in our code to test existing user, we need to make the email consistent.